cli/config.go

```diff
@@ -131,11 +132,21 @@ func (h hypervisor) kernel() (string, error) {
 	return resolvePath(p)
 }
 
+func (h hypervisor) initrd() (string, error) {
+	p := h.Initrd
+
+	if p == "" {
+		return "", nil
+	}
+
+	return resolvePath(p)
+}
+
 func (h hypervisor) image() (string, error) {
 	p := h.Image
 
 	if p == "" {
-		p = defaultImagePath
```

Comment:
You're modifying the default behavior here. I am not against it, but it is worth being mentioned from the commit message I think.

Reply:
Yes that's a good point, there is no reason to maintain default anymore I guess.

Reply:
Yup, I've added description of the removal in the commit message[1]. Thanks! 

[1] https://github.com/kata-containers/runtime/pull/98/commits/a67e5068d5028bddb1e889d5c0efa47d6efcb638